frontend/keyboardNav.js

```diff
@@ -104,15 +104,15 @@ function getRootSelection(dest, store, id) {
     return store.skipWrapper(roots.get(ix + 1));
   }
   return null;
-	}
+}
 
 function getNewSelection(dest: Dest, store: Store): ?ElementID {
   var id = store.selected;
   if (!id) {
     return undefined;
   }
   var node = store.get(id);
-  var pid = store.skipWrapper(store.getParent(id), true);
```

Comment:
Can you help me understand why this call is no longer necessary? What was it doing?